python/semantic_kernel/ai/open_ai/services/open_ai_text_completion.py

```diff
@@ -80,13 +87,17 @@ async def complete_simple_async(
                 f"but logprobs={request_settings.logprobs} was requested",
             )
 
-        openai.api_key = self._api_key
-        if self._org_id is not None:
-            openai.organization = self._org_id
+        openai = self._setup_open_ai()
```

Comment:
Do we need to do this setup for every function call? Can we not do this in the __init__ method?

Reply:
Good point, will move!